src/selectors/profile.js

```diff
@@ -903,6 +927,46 @@ export const getProfileFilterPageDataByTabID: Selector<
   extractProfileFilterPageData
 );
 
+/**
+ * Get the profile filter page data for all the tabs and return a sorted array
+ * of tabs data with their score.
+ */
+export const getProfileFilterSortedPageData: Selector<SortedTabPageData> =
+  createSelector(
+    getProfileFilterPageDataByTabID,
+    getTabToThreadIndexesMap,
+    getThreadActivityScores,
+    (pageDataByTabID, tabToThreadIndexesMap, threadActivityScores) => {
+      const pageDataWithScore = [];
+      // Generate the pageDataWithScore array
+      for (const [tabID, pageData] of pageDataByTabID.entries()) {
+        let tabScore = 0;
+        const threadIndexes = tabToThreadIndexesMap.get(tabID);
+        if (!threadIndexes) {
+          // Couldn't find any thread indexes for the tab. Do not show it.
+          continue;
+        }
+        for (const [threadIndex] of threadIndexes.entries()) {
+          const threadScore = threadActivityScores[threadIndex];
+          if (!threadScore) {
+            throw new Error('Failed to find the thread score!');
+          }
+
+          tabScore += threadScore.boostedSampleScore;
```

Comment:
If I understand properly, when 2 tabs share the same content process (which can happen on Android especially), both tabs will get the score for the full length of the process.
Is that correct ? Is that what we want ? Do you have plan to alleviate this issue ?

I'm fine with landing it like this but it would be good to have some plan to fix it if that's an issue.